Sources/SourceKitLSP/Documentation/DocumentationLanguageService.swift

```diff
@@ -0,0 +1,296 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2024 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import BuildSystemIntegration
+import Csourcekitd
+import Dispatch
+import Foundation
+import IndexStoreDB
+import SKLogging
+import SemanticIndex
+import SwiftExtensions
+import SwiftParser
+import SwiftParserDiagnostics
+
+#if compiler(>=6)
+package import LanguageServerProtocol
+package import SKOptions
+package import SwiftSyntax
+package import ToolchainRegistry
+#else
+import LanguageServerProtocol
+import SKOptions
+import SwiftSyntax
+import ToolchainRegistry
+#endif
```

Comment:
Are all of these imports needed? I wouldn’t expect to need eg. IndexStoreDB, BuildSystemIntegration.

Reply:
You're right. I've removed all the unnecessary imports here.